### ISSUE TITLE
Arrays instantiated without minItems leads to a non-empty array

### ISSUE THREAD

**Issue by limitlis** [tomarad/JSON-Schema-Instantiator#17]
I've recently ran into this issue where instantiating an array lead to non-empty arrays being created.

According to the [spec](http://json-schema.org/latest/json-schema-validation.html#rfc.section.6.12), omitting the **minItems** should be treated as a `minItems: 0`.

Currently this is what you would get.

**Example Schema**
```JSON
    {
      "title": "todo",
      "type": "object",
      "properties": {
        "todos": {
          "type": "array",
          "items": {
            "type": "integer"
          }
        }
      },
      "required": ["todos"]
    };
```

**Expected**
```
{
  'todos': []
}
```
**Actual**
```
{
  'todos': [0]
}
```
I've got a feature branch in my fork with an accompanying test. Will open a PR shortly, but wanted to post here in case there was a reason for this behavior.

Thanks!

**Comment by limitlis** [comment]
The PR I mentioned earlier: https://github.com/tomarad/JSON-Schema-Instantiator/pull/18

**Comment by tomarad** [comment]
Hey!
Thanks for opening this issue.
Will look into it and #13 over the weekend.

Tom

**Comment by limitlis** [comment]
Awesome! Thanks Tom, have a great weekend!

**Comment by tomarad** [comment]
Merged #18. Thanks :)
